syntax/std_os_nonwasm.go

```diff
@@ -58,6 +58,33 @@ func stdOsFile(_ context.Context, v rel.Value) (rel.Value, error) {
 	return rel.NewBytes(f), nil
 }
 
+// stdOsTree returns an array of paths to files within the given directory path.
+func stdOsTree(_ context.Context, v rel.Value) (rel.Value, error) {
+	d, ok := v.(rel.String)
+	if !ok {
+		return nil, errors.Errorf("tree arg must be a string, not %T", v)
+	}
+
+	fs := []rel.Value{}
+	err := filepath.Walk(d.String(), func(path string, info os.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		fs = append(fs, rel.NewTuple(
+			rel.NewStringAttr("name", []rune(info.Name())),
+			rel.NewStringAttr("path", []rune(path)),
+			rel.NewBoolAttr("isDir", info.IsDir()),
+			rel.NewFloatAttr("size", float64(info.Size())),
+			rel.NewFloatAttr("modTime", float64(info.ModTime().UnixNano())),
```

Comment:
for consistency with other stdlib functions
```suggestion
			rel.NewFloatAttr("mod_time", float64(info.ModTime().UnixNano())),
```